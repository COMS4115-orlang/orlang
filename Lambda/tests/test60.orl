let main = 
  let y = 29.1 in
  print_int_endline (fptosi y)
  let y = (10 * 3) in
  print_int_endline (fptosi (sitofp y))
