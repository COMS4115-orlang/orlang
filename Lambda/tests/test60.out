4624633867356078080
